even_fib_nums.js

```diff
@@ -5,22 +5,34 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
+function _sumFibs(maxFibValue) {
   var sum = 0;
-
-  // do your work here
-
+  let fibSeq = [1, 1];
+
+  for (let i = 1; i < fibSeq.length; i++) {
+    let nextSeq = fibSeq[i] + fibSeq[i-1]
+    if (fibSeq[i] <= maxFibValue){
+      fibSeq.push(nextSeq)
+      if (nextSeq%2 === 0){
+        sum += nextSeq;
+      }
+    }
+  }
   return sum;
 }
 
 // bonus round
-function _highestFibonacciNumber (maxFibValue){
+function _highestFibonacciNumber(maxFibValue) {
   var highest = 0;
+  let fibSeq = [1, 1];
 
-  //define your base case, validate your input
-
-
-  //do your work here
+  for (let i = 1; i < fibSeq.length; i++) {
```

Comment:
Same note as above in regards to a for loop that is modifying the array as it's being looped over.